### PR TITLE
Parquet operators test

### LINKED ISSUE #317: Operators acceptance tests for Parquet

### PR DESCRIPTION
**Pull request by sgomezg**
#### Description
Operators test with parquet output. Refactor of test packages to split outputs and operators. More packages is coming :)

It closes #317

#### Reviewer
@arinconstrio  

#### Test
1 test added. All passed

#### Coverage
42.34%.

#### Scalastyle
All test passed.

### CODE REVIEW

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

File: test-at/src/test/scala/com/stratio/sparkta/testat/operators/ISocketOMongoOperatorsIT.scala [hunk 1]
```diff
@@ -59,6 +62,7 @@ class ISocketOMongoOperatorsIT extends MongoEmbedDatabase with SparktaATSuite {
       sendPolicy(PathToPolicy)
       sendDataToSparkta(PathToCsv)
       sleep(PolicyEndSleep)
+      SparktaHelper.shutdown
```
Comment (arinconstrio) [inline test-at/src/test/scala/com/stratio/sparkta/testat/operators/ISocketOMongoOperatorsIT.scala]: Could you move the shutdown method to the ATSuite?

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went KO. Its up to someone to merge.

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went OK. Its up to someone to merge.

**Comment by arinconstrio** [comment]
At the end Jenkis said OK, and this PR is OK for me too.

**Pull request merged**